iree/tools/CMakeLists.txt

```diff
@@ -478,6 +478,21 @@ if(${IREE_BUILD_COMPILER})
     RUNTIME DESTINATION bin
   )
 
+  if(NOT LLD_SYMLINKS_TO_CREATE)
+  set(LLD_SYMLINKS_TO_CREATE
```

Comment:
can drop the darwinnew/darwinold

Reply:
Will do. I copied the link set from `third_party/llvm-project/lld/tools/CMakeLists.txt`